lms/templates/admin/organization/base.html.jinja2

```diff
@@ -0,0 +1,3 @@
+{% extends "lms:templates/admin/base.html.jinja2" %}
+{% block header %}Organization {{ org.id }}{% endblock %}
```

Comment:
Initially I had the tab headers rendered here but now it's each tab that calls the macro.

This base template doesn't add much right now but it useful to have it.